lib/modules/datasource/packagist/schema.ts

```diff
@@ -178,17 +171,19 @@ export const RegistryFile = z.intersection(
 export type RegistryFile = z.infer<typeof RegistryFile>;
 
 export const PackagesResponse = z.object({
-  packages: looseRecord(ComposerReleases),
+  packages: LooseRecord(ComposerReleases).catch({}),
```

Comment:
catch here?

Reply:
Otherwise the whole `PackagesResponse` will fail on empty `packages` field. Previous implementation did this automatically, but it's not always the desired behavior, so I've just ensured each occurrence of older implementation has its catch value.